velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -100,5 +108,16 @@ void registerArrayFunctions() {
   registerArrayCombinationsFunctions<Varchar>();
   registerArrayCombinationsFunctions<Timestamp>();
   registerArrayCombinationsFunctions<Date>();
+
+  registerArrayShuffleFunctions<int8_t>();
```

Comment:
Can we add support for all input types, e.g. array(array(x)), array(map(k, v)), etc?

Reply:
I will leave those nested complex types as a TODO item for now because 1) need to change the simple api code; 2) there are more UDFs to support generic complex types. See https://github.com/facebookincubator/velox/issues/3484